doc/packet_trimming/packet-trimming-design.md

```diff
@@ -351,8 +358,22 @@ In order to use trimming configuration in ACL rule, a dedicated ACL table type m
 
 Existing infrastructure will be reused.
 
-Flex counter groups `port/queue` will be extended with a new SAI attributes  
-`SAI_PORT_STAT_TRIM_PACKETS/SAI_QUEUE_STAT_TRIM_PACKETS` respectively.
+Flex counter groups `port/queue` will be extended with a new SAI attributes:
+* `SAI_QUEUE_STAT_TRIM_PACKETS`
+* `SAI_QUEUE_STAT_DROPPED_TRIM_PACKETS`
+* `SAI_QUEUE_STAT_TX_TRIM_PACKETS`
+* `SAI_PORT_STAT_TRIM_PACKETS`
+* `SAI_PORT_STAT_DROPPED_TRIM_PACKETS`
+* `SAI_PORT_STAT_TX_TRIM_PACKETS`
+
+Flex counter group `switch` will be created to handle switch related statistics:
+* `SAI_SWITCH_STAT_DROPPED_TRIM_PACKETS`
+* `SAI_SWITCH_STAT_TX_TRIM_PACKETS`
+
```

Comment:
Please add a note that when vendor SAI supports these counter attributes, Sonic flexcounter handles the counter poll instead of lua script.

Reply:
@kperumalbfn LUA calculation is not applicable for Switch Level Stats. It is a pure legacy Flex Counter behavior